server/embed/config_logging.go

```diff
@@ -55,6 +57,34 @@ func (cfg *Config) setupLogging() error {
 			}
 		}
 
+		var logRotateConfig *lumberjack.Logger
```

Comment:
This function is already huge. 
Please consider moving it to a separate method.